docs: add auto-lighthouse entry to 'related projects' (#9954)
GoogleChrome/lighthouse: readme.md

## Related Projects
Other awesome open source projects that use Lighthouse.

> * **[auto-lighthouse](https://github.com/TGiles/auto-lighthouse)** - a CLI for crawling a domain, generating mobile and desktop reports, and viewing them automatically after generation.

Reviewer: might want to add this great description to the repo itself @TGiles this sounds awesome, and I'm not really sure what "automated Lighthouse auditing" meant on its own :)

Reviewer: ```suggestion
* **[auto-lighthouse](https://github.com/TGiles/auto-lighthouse)** - a CLI for crawling a domain and generating mobile and desktop reports for each page.
```

minor suggestion since the key features are crawling and mobile+desktop
